src/capture/pcap_middleware/pcap_middleware.h

```diff
@@ -26,49 +26,11 @@
 #ifndef PCAP_MIDDLEWARE_H
 #define PCAP_MIDDLEWARE_H
 
-#include <stdint.h>
-#include <pcap.h>
-
-#include "../../utils/allocs.h"
-#include "../../utils/os.h"
-#include "../../utils/squeue.h"
-#include "../../utils/eloop.h"
-
-#include "../pcap_service.h"
-
-/**
- * @brief Initialises the pcap middleware
- *
- * @param db The sqlite3 db
- * @param db_path The sqlite3 db path
- * @param eloop The eloop structure
- * @param pc The pcap context
- * @return struct middleware_context* the middleware context on success, NULL on
- * failure
- */
-struct middleware_context *init_pcap_middleware(sqlite3 *db, char *db_path,
-                                                struct eloop_data *eloop,
-                                                struct pcap_context *pc);
+#include "../middleware.h"
 
 /**
- * @brief Pcap processor
- *
- * @param context The middleware context
- * @param ltype The packet type
- * @param header The pcap packet header
- * @param packet The pcap packet
- * @param ifname The capture interface
- * @return int 0 on success, -1 on failure
+ * @brief PCAP Middleware.
+ * @author Alexandru Mereacre, Alois Klink
```

Comment:
Same here, we should add a description of what the PCAP middleware does.

Reply:
It stores the raw packet (header + data) in files with metadata in the capture.sqlite

Reply:
Documented in 703faf635ad3cbe868087f843f082bc942129e0f
I also added that is it recommended to run the `cleaner_middleware` too, as the PCAP data stores a lot of bytes.